autotx/tests/agents/token/test_swap_and_send.py

```diff
@@ -20,7 +20,7 @@ def test_auto_tx_swap_and_send_simple(configuration, auto_tx, test_accounts):
     new_wbtc_safe_address = manager.balance_of(wbtc_address)
     new_receiver_wbtc_balance = get_erc20_balance(client.w3, wbtc_address, receiver)
 
-    assert new_wbtc_safe_address == wbtc_safe_address + 0.04
```

Comment:
Shouldn't we have a check that accounts for slippage? It could be `original_amount + 0.04` plus or minus `0.04 * slippage`